README.md

```diff
@@ -32,20 +855,1218 @@ The written tests available in the directory [`test`](./test) achieve a test cov
+        "type": "tuple"
+      },
+      {
+        "internalType": "address",
+        "name": "refundAddress",
+        "type": "address"
+      }
+    ],
+    "name": "deployCreateAndInit",
+    "outputs": [
+      {
+        "internalType": "address",
+        "name": "newContract",
+        "type": "address"
+      }
+    ],
+    "stateMutability": "payable",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      {
+        "internalType": "address",
+        "name": "implementation",
+        "type": "address"
+      },
+      {
+        "internalType": "bytes",
+        "name": "data",
+        "type": "bytes"
+      }
+    ],
+    "name": "deployCreateClone",
+    "outputs": [
+      {
+        "internalType": "address",
+        "name": "proxy",
+        "type": "address"
+      }
+    ],
+    "stateMutability": "payable",
+    "type": "function"
+  }
+]
+```
+
+</details>
+
+## New Deployment(s)
+
+> **Warning**<br>
+> The address `0x0000000000000000000000000000000000000000` is a simple placeholder for now. Do not send any funds there!
+
+We offer two options for deploying [`CreateX`](./src/CreateX.sol) to your desired chain:
+
+1. Deploy it yourself by using one of the pre-signed transactions. Details can be found in the subsequent paragraph.
+2. Request a deployment by opening an [issue](https://github.com/pcaversaccio/createx/issues/new?assignees=pcaversaccio&labels=new+deployment+%E2%9E%95&projects=&template=deployment_request.yml&title=%5BNew-Deployment-Request%5D%3A+). You can significantly reduce the time to deployment by sending funds to cover the deployment cost to the deployer account: [`0x0000000000000000000000000000000000000000`](https://etherscan.io/address/0x0000000000000000000000000000000000000000).
 
-TBD
+TBD (section on pre-signed transactions; we will offer multiple pre-signed transactions with different `gasLimit` levels; will be added after the feedback phase)
 
-## Deployment Approach
+### Contract Verification
 
-TBD
+To verify a deployed [`CreateX`](./src/CreateX.sol) contract on a block explorer, use the following parameters:
 
-## How to Request a Deployment
+- TBD (will be added after the feedback phase)
 
-TBD
+## [`CreateX`](./src/CreateX.sol) Deployments
 
-## Deployments [`CreateX`](./src/CreateX.sol)
+> **Warning**<br>
+> The address `0x0000000000000000000000000000000000000000` is a simple placeholder for now. Do not send any funds there!
 
-TBD
+- EVM-Based Production Networks:
+  - Ethereum: [`0x0000000000000000000000000000000000000000`](https://etherscan.io/address/0x0000000000000000000000000000000000000000)
+- Ethereum Test Networks:
+  - Goerli: [`0x0000000000000000000000000000000000000000`](https://goerli.etherscan.io/address/0x0000000000000000000000000000000000000000)
+- Additional EVM-Based Test Networks:
+  - Optimism Testnet (Goerli): [`0x0000000000000000000000000000000000000000`](https://goerli-optimism.etherscan.io/address/0x0000000000000000000000000000000000000000)
```

Comment:
What do you think about the [JSON file](https://github.com/mds1/multicall/blob/main/deployments.json) approach I used for Multicall3 instead? We can fork the https://www.multicall3.com/ frontend too, that way it's easy to have a table searchable by name or Chain ID. Can also add any other fields/info you want to the JSON and table in the UI if we go that route

(Side benefit is some extra promotion when people [tweet](https://twitter.com/MaximilianFSN/status/1719391868146909191)) out their chain on the website 😅

Reply:
That's actually only a placeholder 😄 - I personally like having everything in one place (in our case in the `README`), but we can the `JSON` file approach as well mirror it later in a UI & add some further infos like `chainId`, link to e.g. EVM Diff etc. So IMO, the `README` should still have it, but without all the additional information. Would you agree with this approach? Also, we can decide on this in 2-3 weeks after the feedback phase.

Reply:
Cool, that sounds good to me! If we stand up a frontend I can manage that part